rtptransceiver.go

```diff
@@ -1,3 +1,4 @@
+//go:build !js
```

Comment:
Is this due to Go version? Does this affect older supported version?

Reply:
looks added by IDE, removed.